caravel/assets/javascripts/SqlLab/components/DataPreviewModal.jsx

```diff
@@ -0,0 +1,58 @@
+import * as Actions from '../actions';
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { Modal } from 'react-bootstrap';
+
+import ResultSet from './ResultSet';
+
+const propTypes = {
+  queries: React.PropTypes.array,
+  actions: React.PropTypes.object,
+  showDataPreviewModal: React.PropTypes.bool,
```

Comment:
this component is very tightly coupled with the app/state (couldn't be reused outside this app really), and this is the key name in the state so I'll keep it the same so it's easy to follow.